hazelcast/src/main/java/com/hazelcast/instance/impl/ClusterTopologyIntent.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.instance.impl;
+
+public enum ClusterTopologyIntent {
+    /**
+     * Hazelcast is not deployed in a managed context
```

Comment:
can you please javaDoc for all enums here, what to do or what not to do next, if hz is not in managed context? What is the consecution of it?

Reply:
added class + values javadoc in https://github.com/hazelcast/hazelcast/pull/21844/commits/d0997b4f066efde9c3686094e114258c079a1702